patches/server/0010-Adventure.patch

```diff
@@ -1110,6 +1110,78 @@ index d63a712126973fd1bea547d30c7d116c622669ee..1f5050e6c1d932aa196ab9524f7f1f9b
          // Spigot start
          if (this.components != null) {
              buf.writeUtf(net.md_5.bungee.chat.ComponentSerializer.toString(components));
+diff --git a/src/main/java/net/minecraft/network/protocol/game/ClientboundSetActionBarTextPacket.java b/src/main/java/net/minecraft/network/protocol/game/ClientboundSetActionBarTextPacket.java
+index 02183c810f9968621b9b20c1f7b54258b620c507..fd885324c2ee33e271592277d8d860e383b30633 100644
+--- a/src/main/java/net/minecraft/network/protocol/game/ClientboundSetActionBarTextPacket.java
++++ b/src/main/java/net/minecraft/network/protocol/game/ClientboundSetActionBarTextPacket.java
+@@ -6,6 +6,7 @@ import net.minecraft.network.protocol.Packet;
+ 
+ public class ClientboundSetActionBarTextPacket implements Packet<ClientGamePacketListener> {
+     private final Component text;
++    public net.kyori.adventure.text.Component adventure$text; // Paper
+ 
+     public ClientboundSetActionBarTextPacket(Component message) {
+         this.text = message;
+@@ -17,6 +18,11 @@ public class ClientboundSetActionBarTextPacket implements Packet<ClientGamePacke
+ 
+     @Override
+     public void write(FriendlyByteBuf buf) {
++        // Paper start
++        if (this.adventure$text != null) {
++            buf.writeComponent(this.adventure$text);
++        } else
++            // Paper end
```

Comment:
```suggestion
+        // Paper end
```

Reply:
oh yep, changed that back in the wrong patch. Done